documentation/TechnicalDesignDocument.tex

```diff
@@ -97,11 +97,11 @@
     Combining these parts one could create a functional view for the user.
     This way a screen can only contain one renderable as well.
     These renderables, mostly applicable to the 3D area, contain entities.
-    Entities could be anything, and for clarification we drew a block entity.
+    Entities could be anything, and for clarification a block entity is drawn.
     All the blocks drawn actually exist out of one entity.
     Out of that entity, three instances are created.
     If you want a different shape, you should add an entity, and create an instance out of that.
-    This system offers a lot of modularity since we can easily add more entities, more screens and more renderables.
+    This system offers a lot of modularity since it is not complicated to add more entities, more screens and more renderables.
```

Comment:
I removed the 'we' words because I don't think it is professional to use this word in a TDD.